With gcc6 there is a space required between string literals
and macro parameters.

--- server/VirtualDrawable.cpp	2016-02-16 03:25:04.000000000 +0100
+++ server/VirtualDrawable.cpp	2016-10-03 15:59:16.995129603 +0200
@@ -27,7 +27,7 @@
 using namespace vglserver;
 
 
-#define CHECKGL(m) if(glError()) _throw("Could not "m);
+#define CHECKGL(m) if(glError()) _throw("Could not " m);
 
 // Generic OpenGL error checker (0 = no errors)
 static int glError(void)
